crates/ruff_linter/src/rules/flake8_pyi/rules/duplicate_union_member.rs

```diff
@@ -93,40 +93,38 @@ pub(crate) fn duplicate_union_member<'a>(checker: &mut Checker, expr: &'a Expr)
         return;
     }
 
-    if checker.settings.preview.is_enabled() {
-        // Mark [`Fix`] as unsafe when comments are in range.
-        let applicability = if checker.comment_ranges().intersects(expr.range()) {
-            Applicability::Unsafe
-        } else {
-            Applicability::Safe
-        };
-
-        // Generate the flattened fix once.
-        let fix = if let &[edit_expr] = unique_nodes.as_slice() {
-            // Generate a [`Fix`] for a single type expression, e.g. `int`.
-            Some(Fix::applicable_edit(
-                Edit::range_replacement(checker.generator().expr(edit_expr), expr.range()),
+    // Mark [`Fix`] as unsafe when comments are in range.
```

Comment:
It seems like https://github.com/astral-sh/ruff/pull/14270 incorrectly removed the fix for non-preview 🤷 So, technically, this PR makes the rule fixable again

Reply:
hehe, onwards and upwards!